config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -2067,10 +2067,10 @@ presubmits:
         - --repo-root=.
         - --gcp-zone=us-central1-b
         - --parallelism=1
-        - --skip-regex=""
-        - '--label-filter=Feature: isSubsetOf { CPUManager, MemoryManager, TopologyManager }'
+        - --focus-regex=\[Feature:CPUManager\]|\[Feature:MemoryManager\]|\[Feature:TopologyManager\]
```

Comment:
Another item we really want to migrate to is label-filter.

@pohly any idea why this doesn't work?

Reply:
How was it not working?

Reply:
My suspicion is that jobs were not being picked by the ginkgo so I tried to match it to a [working example](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-crio-cgroupv1-node-e2e-resource-managers/1943785554705387520) of the job that used the `--focus-regex`. Other thing I noticed in the working job is that it doesnt change the skip regex(seting --skip-regex="" overwrite the default node skips). We can test if this is the issue before changing using the `--focu-regex`